runtime/src/main/java/org/corfudb/runtime/clients/NettyClientRouter.java

```diff
@@ -167,6 +167,12 @@ public NettyClientRouter(@Nonnull NodeLocator node,
         this.parameters = parameters;
         this.eventLoopGroup = eventLoopGroup;
 
+        // Set timer mapping
+        for (CorfuMsgType type : CorfuMsgType.values()) {
+            timerNameCache.put(type,
```

Comment:
timerNameCache could be an ImmutableMap from guava to explicitly show that the map is immutable, that would be helpful for the readers

Reply:
Done.